pynio/service.py

```diff
@@ -53,6 +55,26 @@ def stop(self):
         """ Stops the nio Service. """
         self._instance._get('services/{}/stop'.format(self._name))
 
+    def command(self, *args, **kwargs):
+        '''send a command to the service or to the block.
+        To the service:
+            service.command('command')
+        To a block in the service:
+            service.command(block, 'command')
+
+        kwargs are passed onto the request. Some of use are:
+            data: add data onto the request
+            timeout: set the request timeout
+        '''
+        get = self._instance._get
```

Comment:
I get that this works, but I've never seen this done before. I think it would be easier to follow the code if the methods were called more directly.